glusterd2/servers/sunrpc/server.go

```diff
@@ -132,17 +132,10 @@ func (s *SunRPC) acceptLoop(stopCh chan struct{}, l net.Listener, wg *sync.WaitG
 		"transport": ltype})
 	logger.WithField("address", l.Addr().String()).Info("started server")
 
-	sessions := make([]rpc.ServerCodec, 50)
 	for {
 		select {
 		case <-stopCh:
```

Comment:
doesn't this implementation make this code useless?

Reply:
The select on stopCh is still required to return from the acceptLoop. There is no other return from this function. If stopCh is closed, acceptLoop will terminate, along with all the sessions that are still active.

Reply:
Got the return in the func confused with the return for this infinte loop.